src/main/java/seedu/address/logic/commands/FindCommand.java

```diff
@@ -18,9 +18,10 @@ public class FindCommand extends Command {
 
     public static final String MESSAGE_USAGE = COMMAND_WORD
             + ": Finds all students with keywords in a specific prefix. "
-            + "Prefix supported by this search includes: n/, p/, np/, e/, a/, dt/, t/. "
-            + "The specified keywords are case-insensitive, which displays students as a list with index numbers.\n"
-            + "Parameters: KEYWORD [n/KEYWORD] [p/PHONE] [np/NOK_PHONE] [e/KEYWORD] [a/KEYWORD] [dt/DATE] [t/TAG]\n"
+            + "Prefixes supported by this search includes: n/, p/, np/, e/, a/, dt/, t/. "
+            + "The specified keywords are case-insensitive. Students are displayed as a list with index numbers.\n"
+            + "Parameters: [n/NAME] [p/PHONE] [np/NOK_PHONE] [e/EMAIL] [a/ADDRESS] [dt/DATE] [t/TAG]\n"
```

Comment:
@gnahzeus I feel that maybe this could be rephrased as "Students are displayed in a list with index numbers so when using the `find` command, indicate the index number of the student of interest followed by the corresponding parameter. The specified keyword are case-insensitive and the parameters are as follows:"

What do you think?

Reply:
@seanflyyy The find command does not consist an index number actually. A typical find command for your reference would look like `find n/brad`

Reply:
@gnahzeus oh! thanks for pointing that out. I must have had it confused with another command. In which case, perhaps this can be rephrased as, 

"Prefix supported by this search includes: n/, p/, np/, e/, a/, dt/, t/. " 
+ "Students are displayed in a list with index numbers."
+ "The specified keyword which is after the prefix - `NAME` in the case of n/NAME`"
+ " - are case-insensitive and the acceptable parameters are [n/NAME] [p/PHONE] [np/NOK_PHONE] [e/EMAIL][a/ADDRESS] [dt/DATE] [t/TAG]\n"

What do you think?